valuesets/Valueset-England-ServiceRequestCodes.xml

```diff
@@ -1,6 +1,6 @@
 <ValueSet xmlns="http://hl7.org/fhir">
-    <id value="England-ServiceRequestCategoryCodes" />
-    <url value="https://fhir.nhs.uk/ValueSet/England-ServiceRequestCategoryCodes" />
+    <id value="England-ServiceRequestCodes" />
```

Comment:
Why id/url changed from ServiceRequestCategoryCodes to ServiceRequestCodes? 'Category' has been removed

Reply:
ServiceRequest codes is the slice name for the element. <sliceName value="ServiceRequestCode" />
Changing will align it properly with the Slice name (as previous done in [NHSDigital-ServiceRequest](https://simplifier.net/NHSDigital/NHSDigitalServiceRequest/~xml). 
Also, it will properly differentiate it from a valueset with a similar name: <valueSet value="https://fhir.nhs.uk/ValueSet/England-ServiceRequestCategory"